### PR TITLE
[EN] Changes & Additions to RMoTI#2

### PR DESCRIPTION
**Pull request by oralekin**
- Added predictions explanation
- changed how the tournament is referred to to go with the folder name

### CODE REVIEW

File: wiki/Tournaments/RMoT_Invitational/2/en.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-# RMoT Invitationals 2
+# RMoT Invitational 2
 
-The Rapid Monthly osu! Tournament Invitationals \#2 is a tournament organized and ran by the Rapid Tournaments team.
+The Rapid Monthly osu! Tournament Invitational \#2 is a tournament organized and ran by the Rapid Tournaments team.
 It is an exclusive tournament where only players who reached semi-finals in one of the last 4 RMoT's get invited.
 
-The tournament brackets can be found [on Challonge.](https://challonge.com/rmotinvitational2)
+The tournament brackets can be found [on Challonge.](https://challonge.com/rmotinvitational2) Most of the information here complies with the [forum post](https://osu.ppy.sh/forum/t/589111).
```
Comment (TPGPL) [inline wiki/Tournaments/RMoT_Invitational/2/en.md]: you sure about the dot in the link?

File: wiki/Tournaments/RMoT_Invitational/2/en.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 [![][flag_PL]MagBlood](https://osu.ppy.sh/u/6178640) won the Invitational, beating [![][flag_PL]bsurma](https://osu.ppy.sh/u/509575) in the finals.
```
Comment (TPGPL) [inline wiki/Tournaments/RMoT_Invitational/2/en.md]: i hate those flags in links; at least give a space between flag link and username 🤔

File: wiki/Tournaments/RMoT_Invitational/2/en.md [hunk 3]
```diff
@@ -34,6 +34,12 @@ The RMoT Invitationals are traditionally held over two weekends.
 The Prizes are handed out in the form of Points. every 36 points can be redeemed for a month of osu!supporter. Invitational winners are awarded twice the amount awarded in RMoT's.
 The winner of the Invitational tournament gets 96 points, 2ⁿᵈ place gets 48 points etc..
 
+### Predictions
+
+Anyone was allowed to predict the outcome of the tournament until the starting date. The predictions took place on Challonge and the top three predicters were awarded half the amount of Points awarded in RMoT's.
+
+[![][flag_PL]MagBlood](https://osu.ppy.sh/u/6178640) won the predictions; with [![][flag_PL]bsurma](https://osu.ppy.sh/u/509575) in second place, [![][flag_PL]Malson](https://osu.ppy.sh/u/6047395) in third place.
```
Comment (TPGPL) [inline wiki/Tournaments/RMoT_Invitational/2/en.md]: flags

File: wiki/Tournaments/RMoT_Invitational/2/en.md [hunk 4]
```diff
@@ -53,7 +59,7 @@ The winner of the Invitational tournament gets 96 points, 2ⁿᵈ place gets 48
 * [![][flag_CA]~~Chanigan~~](https://osu.ppy.sh/u/3266120)
```
Comment (TPGPL) [inline wiki/Tournaments/RMoT_Invitational/2/en.md]: flags

File: wiki/Tournaments/RMoT_Invitational/2/en.md (hunk 1 above)
Comment (oralekin) [inline wiki/Tournaments/RMoT_Invitational/2/en.md]: at the end of the sentence? I'm pretty sure but I'll check ASG for any mention of it.

**Pull request merged**